Added ZenFiles to README (#457)
zenml-io/zenml: README.md

| 🗳 **[Vote for Features]** | Pick what we work on next! |
| 📓 **[Docs]** | Full documentation for creating your own ZenML pipelines. |
| 📒 **[API Reference]** | The detailed reference for ZenML's API. |

> | 🗂️️ **[ZenFiles]** | End-to-end fully fledge projects using ZenML. |

Reviewer: ```suggestion
| 🗂️️ **[ZenFiles]** | End-to-end projects using ZenML. |
```

pipeline.run()
```

# 🗂️ ZenFiles

> ZenFiles are production-grade ML use-cases powered by ZenML. They are fully fleshed out, end-to-end, projects that show-case 

Reviewer: ```suggestion
ZenFiles are production-grade ML use-cases powered by ZenML. They are fully fleshed out, end-to-end, projects that showcase 
```

> ZenML's capabilities. They also serve as a template to start similar projects from.

Reviewer: ```suggestion
ZenML's capabilities. They can also serve as a template from which to start similar projects.
```
